From: Luigi Baldoni <aloisio@gmx.com>
Date: 2018-02-11 13:31:28 +0100
Subject: Build PIE binaries

Set lazarus project files so that we produce position
independent executables.

Index: peazip-10.9.0.src/dev/project_pea.lpi
===================================================================
--- peazip-10.9.0.src.orig/dev/project_pea.lpi
+++ peazip-10.9.0.src/dev/project_pea.lpi
@@ -646,6 +646,8 @@
         <DebugInfoType Value="dsDwarf3"/>
       </Debugging>
       <Options>
+        <PassLinkerOptions Value="True"/>
+        <LinkerOptions Value="-pie"/>
         <Win32>
           <GraphicApplication Value="True"/>
         </Win32>
Index: peazip-10.9.0.src/dev/project_peach.lpi
===================================================================
--- peazip-10.9.0.src.orig/dev/project_peach.lpi
+++ peazip-10.9.0.src/dev/project_peach.lpi
@@ -1772,6 +1772,8 @@
         <DebugInfoType Value="dsDwarf3"/>
       </Debugging>
       <Options>
+        <PassLinkerOptions Value="True"/>
+        <LinkerOptions Value="-pie"/>
         <Win32>
           <GraphicApplication Value="True"/>
         </Win32>
